app/src/main/java/com/github/multimatum_team/multimatum/model/Deadline.kt

```diff
@@ -38,7 +49,7 @@ data class Deadline(
     val state: DeadlineState,
     val dateTime: LocalDateTime,
     val description: String = "",
-    val id: String = ""
+    val zoneOffset: ZoneOffset = DEFAULT_TIME_ZONE
```

Comment:
I don't think this should be in the model, if you need the timezone when setting deadlines, you could use `TimeZone.getDefault()`. We will maybe later switch to `ZonedDateTime` but we'll see in a future PR.

Reply:
Idk, I think we need to link deadline to timeZone, but since TimeZone.getDefault return a TimeZone it's a bit annoying to cast. 

And since it's not likely that we'll use the app elsewhere, it should be fine this way